content/projekte/2023-04-01-fl-open-accident-map.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: project
+lab: [flensburg]
+imgname: flensburg/unfallkarte_stadt_flensburg.png
+title: Unfallorte auf einer interaktiven webbasierten Karte
+status: Sucht Mitstreiter
+
+links:
+- url: https://github.com/oklabflensburg/open-accident-map
+  name: GitHub
+- url: https://open-accident-map.de
+  name: Website
+
+collaborators:
+- name: Aurelius Wendelken
+  links:
+  - url: https://twitter.com/oklabflensburg
+    name: twitter
+  - url: https://github.com/oklabflensburg
+    name: github
+
+tags:
```

Comment:
TIL: Wir nehmen als Tag eigentlich immer nur  `Umwelt`, `Politik`, `Gesellschaft` oder `Mobilität`. Dann bekommen die Projektkarten nämlich ein schönes Icon und es wird nicht einfach nur unformatierter Text  dargestellt ([siehe](https://github.com/okfde/codefor.de/blob/main/themes/codefor-theme/layouts/partials/project-preview.html#L9-L16)).
Sollten wir vielleicht mal ändern und das in `category` oder so umbenennen...

Vielleicht wollt ihr das bei euch aber auch schon mal so anpassen, dass ihr da passende Kategorien benutzt?

Reply:
done :pray: